internal/mvcc/kvstore.go

```diff
@@ -300,10 +300,13 @@ func (s *store) Restore(b backend.Backend) error {
 }
 
 func (s *store) restore() error {
-	reportDbTotalSizeInBytesMu.Lock()
 	b := s.b
```

Comment:
I couldn't find a particular reason to procted `b := s.b` with the mutex. Let me know if I overlooked something.